evalml/pipelines/component_graph.py

```diff
@@ -445,7 +450,10 @@ def _transform_features(
         if not fit:
             X_schema = (
                 self._return_non_engineered_features(X).ww.schema
-                if "DFS Transformer" in self.compute_order
```

Comment:
does this mean we were getting the input types wrong for stacked ensemble pipeline with the dfs transformer present?